src/tools/pub/cmd/pub-outdated.md

```diff
@@ -0,0 +1,251 @@
+---
+title: pub outdated
+description: Use pub outdated to help you update your package dependencies.
+diff2html: true
+---
+
+_Outdated_ is one of the commands of the [pub tool](/tools/pub/cmd).
+
+{% prettify nocode tag=pre+code %}
+$ pub outdated [options]
+{% endprettify %}
+
+Use `pub outdated` to identify out-of-date [package dependencies][]
+and get advice on how to update them.
+<span style="color:red">[PENDING: Should we motivate why you want to do this?
+Null safety might be the biggest near-term motivation,
+but are there other major reasons
+that make it worth the effort to update and test your code?]</span>
+
+[package dependencies]: /tools/pub/dependencies
+
+Here's an overview of how you can use `pub outdated`
+to help you update package dependencies:
+
+1. In the top directory of your app or package,
+   **run `pub upgrade`**
+   to create or update the `pubspec.lock` file.
+2. **Run `pub outdated`** and inspect its output.
+3. If the output reports out-of-date dependencies,
+   **update the version dependencies** in `pubspec.yaml` and
+   run `pub upgrade`.
+4. **Run `pub outdated`** to confirm that you've updated the dependencies.
+5. **Test** your app or package to confirm that it still works as expected.
+
+
+## Example
+
+You run `pub outdated` in the top directory of your package or app,
+in the same directory as your package or app's
+[`pubspec.yaml`](/tools/pub/pubspec) file.
+Here's an example of running `pub outdated`:
+
+```terminal
+$ pub outdated
+Package              Current   Upgradable  Resolvable  Latest    
+dependencies: all up-to-date
+
+dev_dependencies    
+build_web_compilers  1.2.3     1.2.3       2.10.0      2.10.0    
+
+transitive dependencies
+_fe_analyzer_shared  -         -           2.0.0       2.0.0     
+analyzer             0.35.4    0.35.4      0.39.5      0.39.5    
+...
+
+9  dependencies are constrained to versions that are older than a resolvable version.
+To update these dependencies, edit pubspec.yaml.
+```
+
+<span style="color:red">[PENDING: Is there a better example we should use?]</span>
```

Comment:
Perhaps the example from the blog post.
I believe it has a bit more going on between the columns.

Reply:
I think I wanted a non-Flutter example (so as to keep dart.dev platform-agnostic), so I picked a random one from site-www. But a more interesting example is probably better.

/cc @mit-mit

Reply:
Maybe an alternate option is to construct your own example; linking to something existing isn't very durable, as it will likely have it's versions rolled and then no longer exhibit the behaviour described in the docs?